test/functional/app/api/test_ussd_api.py

```diff
@@ -127,11 +141,66 @@ def test_golden_path_send_token(mocker, test_client,
 
 
 def test_invalid_service_code(mocker, test_client,
-                                init_database, initialised_blockchain_network, init_seed):
-
+                              init_database, initialised_blockchain_network, init_seed):
     org = OrganisationFactory()
     sender = UserFactory(preferred_language="en", phone=make_kenyan_phone(phone()), first_name="Bob", last_name="Foo",
-        pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
+                         pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
 
-    resp = req("", test_client, sender, '*42*666#')
+    resp = req("", test_client, sender.phone, '*42*666#')
     assert 'END Bonyeza {} kutumia mtandao'.format(valid_service_code) in resp
+
+
+@pytest.fixture(scope='module')
+def create_temporary_user(test_client, init_database, create_organisation):
```

Comment:
This is a fixture so it should be in a `conftest.py` file. Judging from context it looks like it should be in the root one.

Reply:
I grappled with putting it there but then its only used once in this very specific context ? Should such fixtures also be in conftest ?

Reply:
I believe it should be in a `conftest.py` file. But that could be a separate file in this directory, too. If I recall correctly parent dirs are checked in order for that filename to include all fixtures.